docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -107,3 +121,20 @@ For errors like `702 runWeb()] [E] Failed to start server: listen tcp 0.0.0.0:30
 bind: address already in use` gitea needs to be started on another free port. This
 is possible using `./gitea web -p $PORT`. It's possible another instance of gitea
 is already running.
+
+### Git error after updating to a new version of gitea
+
+If the binary file name has been changed during the update to a new version of gitea, 
+git hooks in existing repositories will not work any more. In that case, a git 
+error will be displayed when pushing to the repository.
+
+```
+remote: ./hooks/pre-receive.d/gitea: line 2: [...]: No such file or directory
+```
+
+The `[...]` part of the error message will contain the path to your previous gitea 
+binary.
+
+To solve this, go to the admin options and run the task `Resynchronize pre-receive, 
+update and post-receive hooks of all repositories` to update all hooks to contain
```

Comment:
Please add warning that if a user has custom Git hooks this will overwrite them (I know that the admin dashboard has it, but more warning is better).

Also, another debugging step would be to re-write the authorized key file, as it puts the path to the binary in that file and if the binary name changes then this will also have issues when pushing via SSH.